python/tk_multi_publish2/publish_description_edit.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2017 Shotgun Software Inc.
+#
```

Comment:
Added new class that inherits from QPlainTextEdit and that supports placeholder with a customized touch. This placeholder implementation allows the user to see the summary comments and the placeholder text at the same time for an enhanced clarity and user experience.